cognite_toolkit/_cdf_tk/resource_classes/dataset.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import Field
+
+from .base import ToolkitResource
+
+
+class DataSetYAML(ToolkitResource):
+    external_id: str
```

Comment:
Since you specified max length on all the others, suggest doing so on this one as well.